modules/components/pages/inputs/schema_registry.adoc

```diff
@@ -0,0 +1,285 @@
+= schema_registry
+:type: input
+:status: beta
+:categories: ["Integration"]
+
+// © 2024 Redpanda Data Inc.
+
+
+component_type_dropdown::[]
+
+
+Reads schemas from a schema registry. You can use this connector to extract and backup schemas during a data migration.
+
+Introduced in version 4.32.2.
```

Comment:
We should add the cloud conditions in this PR

Reply:
updated.